docs/source/interfaces/ISIS SANS v2.rst

```diff
@@ -60,10 +60,44 @@ Data Table
+| **SampleTrans**          |   Transmission data file to use.                                                                |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **stp**                  |   Sample scatter period, if not specified all periods will be used (where applicable)           |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **SampleDirect**         |   Direct data file to use                                                                       |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **sdp**                  |   Sample direct period, if not specified all periods will be used (where applicable)            |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **CanScatter**           |   Scatter datafile for can run                                                                  |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **csp**                  |   Can scatter period, if not specified all periods will be used (where applicable)              |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **CanTrans**             |   Transmission datafile for can run                                                             |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **ctp**                  |   Can transmission period, if not specified all periods will be used (where applicable)         |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **CanDirect**            |   Direct datafile for can run                                                                   |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **OutputName**           |   Name of output workspace                                                                      |
++--------------------------+-------------------------------------------------------------------------------------------------+
+| **User File**            |   User file to use for this row, if none is specified default will be used,                     |
+|                          |   if one is specified it will be used instead additionaly overiding any options                 |
+|                          |   set in the gui.                                                                               |
```

Comment:
> User file to use for this row. If specified it will override any options set in the GUI, otherwise the default file will be  used.

Seems clearer to me